docker/nanotube/config/config.toml

```diff
@@ -0,0 +1,53 @@
+# The settings are listed in this file are with their default values.
```

Comment:
Maybe, I'd be possible to reuse the sample main config? This way it will be less confusing for people and the sample config will make more sense and will be open for experimentation.